tools/WinObjC.Tools/vsimporter-templates/WinStore10-Package/Package.targets

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGE TARGETS
+  Description: The target definitions specific to $projectname$
+  For Use By: A VcxProj that depends on the $projectname$ package
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\$(MSBuildThisFileName).Consumption.targets" Condition="Exists('$(MSBuildThisFileDirectory)\$(MSBuildThisFileName).Consumption.targets')" />
```

Comment:
Perhaps I don't understand what Package.targets is here but I thought this is what is going to be the consumption targets anyway. It seems you are making another level of indirection and i"m not sure why.

---
In reply to: [100941520](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100941520) [](ancestors = 100941520,100941133)